node/bft/examples/simple_node.rs

```diff
@@ -129,19 +129,20 @@ pub async fn start_bft(
         Arc::new(BFTMemoryService::new()),
         BatchHeader::<CurrentNetwork>::MAX_GC_ROUNDS as u64,
     );
-    // Initialize the gateway IP and dev mode.
-    let (ip, dev) = match peers.get(&node_id) {
-        Some(ip) => (Some(*ip), None),
-        None => (None, Some(node_id)),
+    // Initialize the gateway's storage mode.
```

Comment:
nit: the IP is also established here

Reply:
https://github.com/ProvableHQ/snarkOS/pull/3513/commits/6732a2499d4955399402d43159f02ab05854efbb